tests/compile-fail/modulo_one.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+#![deny(modulo_one)]
+
+fn main() {
```

Comment:
What kind of passing tests should there be? The only one that springs to mind is:

```10 % 2```

Or something similar.